SIMS/SIMS/Inventory.cs

```diff
@@ -0,0 +1,122 @@
+namespace SIMS;
+
+public class Inventory
+{
+    // Products property
```

Comment:
no need for comments, the code should be self-explanatory. Remove them from your code please.